vernac/g_vernac.mlg

```diff
@@ -183,10 +183,11 @@ GRAMMAR EXTEND Gram
   subprf:
   [ [ s = BULLET -> { VernacBullet (make_bullet s) }
     | "}" -> { VernacEndSubproof }
+    | IDENT "abstract"; ":"; "{" -> { VernacSubproof AbstractSubproof }
```

Comment:
This layer has access to `tclABSTRACT`, right?  It could be
```suggestion
    | IDENT "abstract"; ":"; "{" -> { VernacSubproof (WithTacticSubproof "abstract" (fun tac -> tclABSTRACT None tac))  }
```
or similar, right?

Reply:
I think the ast should generally avoid containing closures